container_linux.go

```diff
@@ -306,5 +306,11 @@ func (c *linuxContainer) currentState() (*State, error) {
 	for _, ns := range c.config.Namespaces {
 		state.NamespacePaths[ns.Type] = ns.GetPath(c.initProcess.pid())
 	}
+	for _, nsType := range configs.NamespaceTypes() {
```

Comment:
I though about it. But for me it leads to unnecessary complexity because of array-ness of Namespaces. Maybe you can propose something.